### PR TITLE
update MANIFEST.in

### LINKED ISSUE #6295: Update MANIFEST.in
Some paths are out of date after examples re-org and bkcharts split

### PR DESCRIPTION
**Pull request by bryevdv**
issues: fixes #6295

Update `MANIFEST.in` in light of recent examples re-orgs and `bkcharts` split. 

The also changes to include the docs in the source tarball.

### CODE REVIEW

**Comment by bryevdv** [comment]
Suggest just reviewing the new file as a whole: 

https://github.com/bokeh/bokeh/blob/67d6964f9805e5778151b1d6a8874248adfde4f3/MANIFEST.in

File: MANIFEST.in [hunk 1]
```diff
@@ -1,21 +1,28 @@
...
-prune examples/compat/pandas/*.html
-prune examples/compat/seaborn/*.html
+
+# Docs
+graft sphinx
+prune sphinx/_build
+
+# Examples
+recursive-include examples *.py *.ipynb *.md
 prune examples/embed/*.html
```
Comment (mattpap) [inline MANIFEST.in]: Why do we have to prune `*.html` files if they aren't included in the line above?

**Comment by bryevdv** [comment]
@mattpap I think you are right, here is a gist of the tarball contents:

https://gist.github.com/bryevdv/71f399335634be28bc60e94321fb34b4

It looks like no HTML files under examples are included at all. But some really should be, which presumably is the intent behind selectively pruning. I will add `*.html` to the `recursive-include` and then merge.

**Comment by bryevdv** [comment]
Honestly if `sdist` is run from a clean tree then it should just keep all `.html` files under examples but seems good to be defensive too.

**Comment by bryevdv** [comment]
Here is output with the change:

https://gist.github.com/bryevdv/51b360ec97e24a10907cb43ea42748c4

Which looks better / correct to me.

**Comment by mattpap** [comment]
@bryevdv, `bokehjs/build/tree_ts` should be excluded. This is an intermediate build directory, not very useful without additional compilation steps.

**Comment by bryevdv** [comment]
@mattpap New gist

https://gist.github.com/bryevdv/67bbb1577e45d1bac74a7b8dee972270

**Comment by mattpap** [comment]
> New gist

Is that before or after "more tweaks", because `tree_ts` is still there.

**Comment by bryevdv** [comment]
Weird. I just pruned all of `tree` and `tree_ts`. The point of an sdist for Bokeh is to have only final pre-built artifacts. New gist:

https://gist.github.com/bryevdv/f3d39706feeec6ef5b9ca442122b7f96

**Pull request merged**